.env.sample.holesky

```
@@ -84,6 +84,9 @@ LIGHTHOUSE_CHECKPOINT_SYNC_URL=https://checkpoint-sync.holesky.ethpandaops.io/
 # MEV-Boost docker container image version, e.g. `latest` or `1.6`.
 #MEVBOOST_VERSION=
 
+# MEV-Boost docker container image name, e.g. flashbots/mev-boost.
+#MEVBOOST_IMAGE=
```

Should we have it commented like this in .env.mainnet.sample, and here once tested, will it shortly be this?: 
```suggestion
MEVBOOST_IMAGE=obolnetwork/mev-boost
```

---

Hmm, so we remove the default altogether?